src/components/AnnouncementCard.tsx

```diff
@@ -57,10 +55,16 @@ export const AnnouncementCard: React.FC<Announcement> = ({
       </div>
     </>
   );
-  return imageSrc ? (
+  return (
     <>
```

Comment:
⛏️ Is returning only the AnnoucementCardCover component inside a fragment different than just returning the component itself? It seems like there doesn't need to be a fragment here, but I see that the other branch returns one so I can be persuaded